src/ec/packages/ec-component-blockquote/blockquote.story.js

```diff
@@ -8,17 +9,49 @@ import data from '@ecl/ec-specs-blockquote/demo/data';
 import blockquote from './ecl-blockquote.html.twig';
 import notes from './README.md';
 
+// Labels for the group ids
+const optionalGroupId = 'Optional elements';
+const requiredGroupId = 'Mandatory elements';
+const useCasesGroup = 'Use cases';
+
+// Preserve the adapted specs.
+const defaultData = { ...data };
+
+// declare your toggled data
+const authorBtnToggler = () => {
+  defaultData.author = defaultData.author ? false : data.author;
+};
+
+const formatQuote = passeddata => {
+  // declare your knobs when you format your data
+  const citation = passeddata.citation
+    ? text('Citation', passeddata.citation, requiredGroupId)
+    : false;
+
+  const author = passeddata.author
+    ? text('Author', passeddata.author, optionalGroupId)
```

Comment:
@Joosthe . where did you see that the author  is an optional parameter?
https://ec.europa.eu/component-library/ec/components/blockquote/usage/
Due to this also the check you added in the template i think is not ok, this falls into the other track about enforcing some check on the input data.